internal/builders/go/main_test.go

```diff
@@ -237,35 +237,36 @@ func Test_runBuild(t *testing.T) {
 			workingDir: "./valid/path/",
 		},
 		// Below are the same tests we do in pkg/config_test.go
+		// TODO: Do they provide additional value then? Can we remove them instead of exporting errors?
```

Comment:
good point. I think they are used in main_test.go, and it's the only reason they are public. A while back, we discussed having all the files in the same folder, in which case we could make them private.

Reply:
The only difference I can tell is that the config is passed through `runBuild` in `main_test.go`  and `ConfigFromFile` (which is called by `runBuild`) in `config_test.go`. Maybe these tests in `main_test.go` are the broader tests?

I think it's fine to leave this as is for now and tackle it in a later refactor.